docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -282,11 +282,44 @@ be sampled such that:
 	# (s is the shape parameter)
 	response_time_distribution = lognorm(s=sigma, scale=median)
 
-**Coverage Data**
+**Population Coverage Data**
+
+The coverage algorithm for vitamin A fortification should follow the same approach described in this concept model document for folic acid fortification (see `Population Coverage Data`_).
+
+.. list-table:: Means and 95% CI's for existing population coverage of vitamin A fortification (% of total population)
+  :widths: 5 5 5 5
+  :header-rows: 1
+
+  * - Location
+    - :math:`a` = Eats fortified vehicle
+    - :math:`b` = Eats fortifiable vehicle
+    - :math:`c` = Eats vehicle
+  * - Ethiopia
+    - 1.0 (see instructions for folic acid parameter)
+    - 44 (34, 54)
+    - 55 (45, 65)
+  * - India
+    - 24.3 (21.1, 27.9)
+    - 89.4 (87.0, 91.8)
+    - 100 (100, 100)
+  * - Nigeria (Kano)
+    - 7.6 (5.9, 9.4)
+    - 35.9 (32.7, 39.1)
+    - 98.4 (97.6, 99.3)
+  * - Nigeria (Lagos)
+    - 7.2 (5.5, 8.9)
+    - 22.7 (19.9, 25.5)
+    - 98.6 (97.8, 99.3)
+
+For all values other than :math: `a` for Ethiopia, use a Beta distribution 
+with mean equal to the central estimate, and variance equal to the variance of 
+a normal distribution with the same mean and 95% confidence interval. For the :
+math: `a` value for Ethiopia, follow the corresponding instructions in the `
+Population Coverage Data`_ section for folic acid.
```

Comment:
Nathaniel didn't actually tell us how to get these parameters.

Reply:
![image](https://user-images.githubusercontent.com/54918472/77187003-0c74ba00-6a91-11ea-97cc-95b8fa597fb0.png)

This is what I was referring to. Is this insufficient?

Reply:
I can add a hyperlink so it's more clear if this info is what you need!

Reply:
Oh, you have different parameter values in your table than were in the folic acid fortification table, so I assumed you wanted to use different beta distributions.  I thought @NathanielBlairStahn computed them from the values in the table.  Is that not true?

Reply:
The Ethiopia "a" for vitamin A is the only value that needs to use the beta distribution method. The Ethiopia "a" value for vitamin A is the same value for Ethiopia "a" value for folic acid, so they will also use the same distribution.

Reply:
I'll add more detail to the text here to help clarify.

Reply:
@collijk Does the new commit clarify things? See my comments below.